src/request_handler.h

```diff
@@ -16,6 +16,35 @@ class Request {
     public:
         static std::unique_ptr<Request> Parse(const std::string& raw_request);
 
+        /*
+         * Mutators
+         */
+
+        void set_uri(const std::string &uri)
```

Comment:
Just a style nitpick, but non-accessors are named in `UpperCamelCase`, and we put `{` on the same line
as the function definition. Feel free to not change this though haha.